.github/workflows/bundlewatch.yml

```diff
@@ -47,7 +47,9 @@ jobs:
       working-directory: client-admin
       run: |
         npm install
-        npm run build:webpack
+        cp polis.config.template.js polis.config.js
+        npm run build:prod
+        mv dist/static/js/admin_bundle.*.js dist/static/js/admin_bundle.xxxxxxxx.js
```

Comment:
@patcon Do you remember what the intention here was? Or was this just stubbed out and not revisited?

I know that in some places we timestamp file paths for caching, so I'm not sure if this was intended to fit into that scheme, or if it's something that didn't get finished.

Thanks

Reply:
Ack, sorry to leave this hanging! I was thinking the xxxxx was permanent solution.

It was because the bundlesize tool depends on files having the same filename between runs. So this just removes to the dynamic hash. This helps compare them and offer helpful feedback about changes:

console log with link: https://github.com/compdemocracy/polis/actions/runs/3079836348/jobs/4976514743#step:9:17
[<img width="1040" alt="Screen Shot 2022-09-30 at 4 54 40 PM" src="https://user-images.githubusercontent.com/305339/193309444-582d3936-7c5c-4438-958f-e5907412b700.png"><sup>link to page</sup>](https://service.bundlewatch.io/results?d=details%7CrepoOwner%7Ccompdemocracy%7CrepoName%7Cpolis%7CrepoCurrentBranch%7Ccolinmegill/node-18-upgrade%7CrepoBranchBase%7Cdev%7CcommitSha%7C5f7b80f26b428b033643f237560fe818c31959d0%7Cresults%7Cstatus%7Cpass%7CfullResults%7CfilePath%7Cclient-admin/dist/admin_bundle.js%7Csize%7CbaseBranchSize%7CmaxSize%7Cclient-participation/dist/cached/xxxxxxxxx/js/polis.js%7Cclient-participation/dist/cached/xxxxxxxxx/js/vis_bundle.js%7Csummary%7CEverything%2Bis%2Bin%2Bcheck%2B(%252B0B,%2B-67B,%2B-0.0%2525)%5E3SE9%7C3SE9%7C3Y80%7C39L4%7C39MZ%7C3AIO%7C48FE%7C48FE%7C4E0W%5E%5E$0%7C$1%7C2%7C3%7C4%7C5%7C6%7C7%7C8%7C9%7CA%5D%7CB%7C$C%7CD%7CE%7C@$F%7CG%7CC%7CD%7CH%7CO%7CI%7CP%7CJ%7CQ%5D%7C$F%7CK%7CC%7CD%7CH%7CR%7CI%7CS%7CJ%7CT%5D%7C$F%7CL%7CC%7CD%7CH%7CU%7CI%7CV%7CJ%7CW%5D%5D%7CM%7CN%5D%5D)

When PRs are created off branches in the upstream repo instead of forks, a check runs that links it from the PR (rather than hiding it in console logs).

Reply:
Thanks for getting back to us on this @patcon.

Am I correct in understanding that that this only affects the bundlewatch workflow, and not the production builds? I see the value of being able to present diffs with bundlewatch, but not at the expense of our current caching strategy.

One way or the other, we should have this documented in comments to avoid confusion in the future.

Thanks again

Reply:
no prob! but yes, this only happens in the bash line of the workflow, and nowhere else. so no effect on prod builds :)

This should be confirmed with `git grep .xxxxxxxx.js` and that should be the only place it happens

Reply:
OK; Great. Thank you for confirming. I'd appreciate a line comment about this so future devs know why it's happening, but otherwise looks good.